tools/bin/commands/dev

```diff
@@ -24,4 +111,13 @@ dev::run() {
         local pod=$(oc get -o name pod -l component=syndesis-${name})
         oc port-forward ${pod//*\//} 5005:5005
     fi
+
+    if [ -n "$(hasflag --cookie)" ]; then
+        if [ -n "$(hasflag --refresh)" ]; then
+             dev::create_api_cookie
+        fi
+        echo $COOKIE1
```

Comment:
you should move this up and add an exit to in `--debug` as both options are orthogonal (as `oc portfoward` also spits on standardout and it doesn't make sense to allow both).